dotnet-consumer-projects.md

```diff
@@ -18,7 +18,7 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
  * [Smuxi IM](https://github.com/meebey/smuxi) - free, distributed and user-friendly IRC / Twitter / XMPP / Campfire / JabbR client for Linux, Windows and Mac OS X 
 
 ## Utilities 
- * [Gmail Notifier Plus](https://github.com/shellscape/Gmail-Notifier-Plus) - An app to monitor multiple Gmail and Google Apps for Work Gmail accounts from a single, compact point, sitting in Windows 7/8/8.1 taskbar.
+ * [Gmail Notifier Plus](https://github.com/shellscape/Gmail-Notifier-Plus) - An app to monitor multiple Gmail accounts from a single, compact point, sitting in Windows 7/8/8.1 taskbar.
```

Comment:
At https://github.com/shellscape/Gmail-Notifier-Plus, the application is described as an app "to monitor multiple Gmail and G Suite Gmail accounts". It would be best to preserve that wording in the description, @grant.

Your Siberian Husky is really beautiful, by the way.

Reply:
Disclaimer: I work for G Suite.

Gmail on G Suite domains is a subset of Gmail.
From a Google branding perspective, "G Suite Gmail" is discouraged. Please say Gmail.

Reply:
OK. Thanks, @grant. I'll merge your PR.